themes/brainy/README.md

```diff
@@ -1,8 +1,35 @@
 # Brainy theme
 
 Simple colorful terminal prompt theme (inspired by a number of themes).
+Supported on all operating systems
+In constant maintenance and improvement
 
-![alt text](https://www.lfsystems.xyz/img/brainy-Arch.png)
```

Comment:
Thanks - I like the idea of not adding images to the repo, to keep it small. When someone clones Bash-it, they're interested in the features, and don't need/want to download several megabytes of example images for the themes.

Can you please remove the 2.1 MB animated Gif from the PR? This is unnecessary bloat, and mainly increases the size of the repo without providing a lot of value to most people.

Feel free to add the image to the project's Wiki - we have a theme page there that contains screenshots.